developer/cmdlet/adding-user-messages-to-your-cmdlet.md

```diff
@@ -256,11 +271,26 @@ WriteProgress(pr);
     ```
 
 ## See Also
+
  [Create a Cmdlet that Modifies the System](./creating-a-cmdlet-that-modifies-the-system.md)
- [How to Create a Windows PowerShell Cmdlet &#91;delete&#93;](http://msdn.microsoft.com/en-us/0d721742-c849-4d0d-964f-78ddd9cd258c)
- [How to Create a Windows PowerShell Cmdlet &#91;delete&#93;](http://msdn.microsoft.com/en-us/0d721742-c849-4d0d-964f-78ddd9cd258c)
+
+
+ [How to Create a Windows PowerShell Cmdlet &#91;delete&#93;](https://msdn.microsoft.com/en-us/0d721742-c849-4d0d-964f-78ddd9cd258c)
```

Comment:
These links need to be resolved to the markdown file in this docset. I have a CSV list of GUIDs to filenames.